src/core/lib/surface/init.cc

```diff
@@ -122,6 +123,7 @@ void grpc_shutdown_internal_locked(void)
     grpc_core::ExecCtx exec_ctx(0);
     grpc_iomgr_shutdown_background_closure();
     grpc_timer_manager_set_threading(false);  // shutdown timer_manager thread
+    // Note: use AresShutdown() when we remove this function.
```

Comment:
`// TODO(yijiem): `. This will help us track work, and batch them with the current domain owner.

Reply:
removed